### PR TITLE
Remove duplicated navigation bar items for properties with getters and setters

### PR DESCRIPTION
**Pull request by vasily-kirichenko**
This fixes #1 and #2 from https://github.com/Microsoft/visualfsharp/issues/2132 and adds accessibility aware glyphs: 

image

### CODE REVIEW

File: src/fsharp/vs/ServiceNavigation.fs [hunk 1]
```diff
@@ -191,23 +193,42 @@ module NavigationImpl =
...
+                             [ createMember(id, MethodDecl, GlyphMajor.Method2, ty.Range, access) ]
+                         | SynMemberDefn.NestedType _ -> failwith "tycon as member????" //processTycon tycon                
+                         | SynMemberDefn.Interface(_, Some(membs), _) ->
+                             processMembers membs |> snd
+                         | _ -> [] 
+                     // can happen if one is a getter and one is a setter
+                     | [SynMemberDefn.Member(memberDefn=Binding(headPat=SynPat.LongIdent(lid1, Some(info1),_,_,_,_)) as binding1)
+                        SynMemberDefn.Member(memberDefn=Binding(headPat=SynPat.LongIdent(lid2, Some(info2),_,_,_,_)) as binding2)] ->
+                         // ensure same long id
+                         assert((lid1.Lid,lid2.Lid) ||> List.forall2 (fun x y -> x.idText = y.idText))
```
Comment (forki) [inline src/fsharp/vs/ServiceNavigation.fs]: What is the assert doing here? Are we just throwing if this is not fulfilled?

Comment (vasily-kirichenko) [inline src/fsharp/vs/ServiceNavigation.fs]: I've ported this code from here https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/vs/ServiceParseTreeWalk.fs#L418-L454 I think that if this assertion violated, it means that something very wrong happened with the parser and we must see it in debug configuration.

Comment (forki) [inline src/fsharp/vs/ServiceNavigation.fs]: Ok

File: src/fsharp/vs/ServiceNavigation.fs [hunk 2]
```diff
@@ -26,10 +26,10 @@ type FSharpNavigationDeclarationItemKind =
 
 /// Represents an item to be displayed in the navigation bar
 [<Sealed>]
-type FSharpNavigationDeclarationItem(uniqueName: string, name: string, kind: FSharpNavigationDeclarationItemKind, glyph: GlyphMajor, range: range, bodyRange: range, singleTopLevel:bool) = 
+type FSharpNavigationDeclarationItem(uniqueName: string, name: string, kind: FSharpNavigationDeclarationItemKind, glyph: GlyphMajor, range: range,
```
Comment (vasily-kirichenko) [inline src/fsharp/vs/ServiceNavigation.fs]: I'm for 120 or maybe 140 characters lines 👍 

Maybe we should add this code style rule to devguide.md

Comment (cartermp) [inline src/fsharp/vs/ServiceNavigation.fs]: I definitely support that change.

**Pull request merged**